wdl/tasks/PBUtils.wdl

```diff
@@ -258,6 +258,48 @@ task ExtractHifiReads {
     }
 }
 
+task RemoveKinetics {
+    input {
+        File bam
+        String prefix = "no_kinetics"
+
+        RuntimeAttr? runtime_attr_override
+    }
+
+    Int disk_size = 2*ceil(size(bam, "GB"))
+
+    command <<<
+        set -euxo pipefail
+
+        python3 /usr/local/bin/remove_kinetics.py -o ~{prefix} ~{bam}
```

Comment:
Does the new bam need to be re-indexed?
If so, makes sense to index it here in stead of doing that in a separate task.

Reply:
And on a 2nd thought, does the PBI need to be re-generated too?